Organization/Organization/DetailViewController.h

```diff
@@ -1,9 +1,9 @@
-
-
 #import <UIKit/UIKit.h>
-@class Employee;
 
+@class Employee;
 
 @interface DetailViewController : UIViewController
-    @property (strong, nonatomic) Employee* detailItem;
+
+@property (strong, nonatomic) Employee *detailItem;
```

Comment:
You don't own this Employee object in this place, so it should be weak, not strong.